source/_templates/header.html

```diff
@@ -41,8 +41,8 @@
             <a class="ods__documentation-header-nav-item-active" href="{{ pathto(master_doc) }}">FAQ</a>
             <a href="/widgets/">Widgets</a>
             <a href="/{{ language }}/apis/">APIs</a>
-            <a href="/tutorials/{{ language }}/">{{ gettext('Tutorials') }}</a>
+            <a href="/tutorials/{{ language }}/">Academy</a>
```

Comment:
I think this should be a link to `{{ gettext('https://academy.opendatasoft.com/page/homepage') }}` as well

Reply:
Yep, it's what I have in local… :p